spec/mailers/oral_history_delivery_spec.rb

```diff
@@ -16,7 +16,7 @@
     end
 
     let!(:work) do
-      create(:oral_history_work, published: true).tap do |work|
+      create(:oral_history_work, :published).tap do |work|
         work.members = members
         work.representative =  members[2]
```

Comment:
I see this pre-dates this PR, so we don't have to and probably shouldn't fix it in this PR, but seeing this.. I really don't love setting the representative to `members[2]`, like why do we care about the 3rd element of the members list being the representative, why is that our test condition?  The test setup should be more clear about the intent of the scenario being set up. If we care about a specific thing being set as the representative, it should be named and referred to by a name explaining it's significance.

Reply:
Good call -- fixed